publication-commons/src/test/resources/features/AnonymousAccessRights.feature

```diff
@@ -0,0 +1,59 @@
+Feature: Anonymous access rights
+
+  Notes:
+    1. In the following, when are referring to a publication that is published,
+       we mean that is has status PUBLISHED.
+
+  Background:
+    Given that there is a database "PUBLICATIONS" with publications
+    And that DynamoDBPublicationService exists
+    And the DynamoDBPublicationService has a READ method
+    And the READ method takes as parameter a non empty username
+    And the READ method takes as parameter a non empty publication ID
+    And the DynamoDBPublicationService has an UPDATE method
+    And the UPDATE method takes as parameter a non empty username
+    And the UPDATE method takes as parameter a non empty publication ID
+    And the DynamoDBPublicationService has an DELETE method
+    And the DELETE method takes as parameter a non empty username
+    And the DELETE method takes as parameter a non empty publication ID
+    And the DynamoDBPublicationService has a CREATE method
+    And the CREATE method takes as parameter a non empty publication object
+
+
+   Scenario: Anonymous user tries to read published material
```

Comment:
Formatting is weird in the document, occasionally inserting two spaces after the Gherkin keyword